sympy/stats/frv_types.py

```diff
@@ -16,7 +16,6 @@
 RobustSoliton
 """
 
-
```

Comment:
I think that this change should be removed unless it makes better formatting style.
(I'd read PEP-8 if it says about spacing between module docstring and code)

Reply:
Sorry, i fixed it